deepchem/trans/transformers.py

```diff
@@ -1577,7 +1576,8 @@ def transform(self, dataset, parallel=False, out_dir=None, **kwargs):
 
     Returns
     -------
-    `Dataset` object that is transformed.
+    DiskDataset or NumpyDataset
+      `Dataset` object that is transformed.
```

Comment:
Are those really the only ones it works with?  I looked at the code and didn't see any reason it wouldn't work with an ImageDataset (or other dataset classes we might create in the future).

Reply:
This is a return type. I seem that this function returns DiskDataset or NumpyDataset.